src/Tribe/Plugins_List.php

```diff
@@ -0,0 +1,126 @@
+<?php defined( 'WPINC' ) or die;
+
+if (! class_exists( 'Tribe__Plugins_List' ) ) {
+	/**
+	 * A list of Tribe's major plugins. Useful when encouraging users to download one of these.
+	 */
+	class Tribe__Plugins_List {
```

Comment:
No no, not when the plugins are active, just avoid one more class by registering the list of plugins inside of the `Dependency` class, the one you created on this PR.